src/data_cell/graph_datacell_test.cpp

```diff
@@ -0,0 +1,67 @@
+
+// Copyright 2024-present the vsag project
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "graph_datacell.h"
+
+#include "catch2/catch_template_test_macros.hpp"
+#include "fmt/format-inl.h"
+#include "graph_interface_test.h"
+#include "io/io_headers.h"
+using namespace vsag;
+
+template <typename IOTemp>
+void
+TestGraphDataCell(const nlohmann::json& graph_param,
```

Comment:
create an alias of `nlomann::json`, njson or some other name, avoiding third-party dependency in too many files

Reply:
done, but rename operator at other files may in another pull request